File: app/src/main/java/com/yatatsu/passiveviewsample/data/model/Repository.java
package com.yatatsu.passiveviewsample.data.model;


public class Repository {
    public String name;
    public String fullName;
    public String description;
    public String htmlUrl;
    public int stargazersCount;
    public int watchersCount;

    public Repository(String name,
                      String fullName,
                      String description,
                      String htmlUrl,
                      int stargazersCount,
                      int watchersCount) {
        this.name = name;
        this.fullName = fullName;
        this.description = description;
        this.htmlUrl = htmlUrl;
        this.stargazersCount = stargazersCount;
        this.watchersCount = watchersCount;
    }

    public Repository() {

    }


}

File: app/src/main/java/com/yatatsu/passiveviewsample/data/model/Repository.kt
package com.yatatsu.passiveviewsample.data.model


class Repository {
  var name: String? = null
  var fullName: String? = null
  var description: String? = null
  var htmlUrl: String? = null
  var stargazersCount: Int = 0
  var watchersCount: Int = 0

  constructor(name: String,
      fullName: String,
      description: String,
      htmlUrl: String,
      stargazersCount: Int,
      watchersCount: Int) {
    this.name = name
    this.fullName = fullName
    this.description = description
    this.htmlUrl = htmlUrl
    this.stargazersCount = stargazersCount
    this.watchersCount = watchersCount
  }

  constructor() {

  }

}
